src/Performance/Minibench.purs

```diff
@@ -35,6 +40,45 @@ withUnits t
   | t < 1.0e9 = toFixed (t / 1.0e6) <> " ms"
   | otherwise = toFixed (t / 1.0e9) <> " s"
 
+-- | When running `bench` or `benchAff`, this is the number of samples that will
+-- | be used.
+defaultNumSamples :: Int
+defaultNumSamples = 1000
+
+-- | The `Results` type is a `Record` newtype that contains information about
+-- | the timings for the test run.
+newtype Results = Results
+  { mean :: Number
+  , stddev :: Number
+  , min' :: Number
+  , max' :: Number
+  }
+
+instance showResults :: Show Results where
```

Comment:
I don't think we should use `Show` for this. Can you please turn this into a function called something like `displayResults` instead.